samples/ChangeFeedMigrationTool/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.DocumentDB" version="1.14.1" targetFramework="net47" />
-  <package id="Microsoft.Azure.DocumentDB.ChangeFeedProcessor" version="1.0.0" targetFramework="net47" />
+  <package id="Microsoft.Azure.DocumentDB" version="1.17.0" targetFramework="net47" />
```

Comment:
@rnagpal @ealsur I think 4.7 is not a bad think. At least we can merge this PR and add one more with fix to 4.6.1, because from my perspective 4.7 or 4.6.1 is out of scope for this PR.